### DISCUSSION TITLE
Alient Motion Sensor moves it's postion towards right side on level change

### DISCUSSION THREAD

**Discussion by PanmanS** [FakeFishGames/Barotrauma#15318] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

When changing level, alient motion sensors move towards right for no reason.
Before level change
![image](https://github.com/user-attachments/assets/cf5abc8e-9fb9-4f10-9959-d3a3ef9a364e)

After level change
![image](https://github.com/user-attachments/assets/315cd7bb-e5f8-4ac1-b258-622fa130944e)

### Reproduction steps

1. download this submarine and start campaign
[Crystal.zip](https://github.com/user-attachments/files/18052589/Crystal.zip)
2, screenshot position
3. proceed to next level and compare

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by SydWad** [comment]
I've tested and it appears to be related to items that are `<Body bodytype="Static"/>` opposed to `<StaticBody />` and then are scale down or up past their default scale value, and then go through a loading transition.
Happens with alienmotionsensor and aliengenerator_new from what ive tested so far.
![image](https://github.com/user-attachments/assets/67aab65e-9b8f-4d55-9f33-bcba46fb96fb)

**Comment by Regalis11** [comment]
Duplicate of https://github.com/FakeFishGames/Barotrauma/discussions/15132. We've implemented a fix for this in the unstable version of the game, and the fix will be included in the next update as well. Sorry for the inconvenience!
